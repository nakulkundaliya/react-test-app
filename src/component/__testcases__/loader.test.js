import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../loader';
let wrapped = shallow(<Loader />);
describe('render loader component', () => {
    it('should render the Loader component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });
});