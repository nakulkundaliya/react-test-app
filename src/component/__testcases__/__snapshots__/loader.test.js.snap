// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render loader component should render the Loader component correctly 1`] = `ShallowWrapper {}`;
